rest-doc/examples/SupplierAddress.json

```diff
@@ -1,35 +1,23 @@
 {
-  "id":"3",
-  "salutation":null,
-  "name1":null,
-  "type":"default",
+  "addressId":"central_office",
```

Comment:
i thought the mapping of address was also changed to not work n2m, but use numeric key and have supplierId and type as fields?